### ISSUE TITLE
(Feature): Set Curving of Node Connectors

### ISSUE THREAD

**Issue by MKilo24** [NatronGitHub/Natron#780]
### Make sure to follow our issue report guidelines

- [X] I have read the [feature request guidelines](https://hackmd.io/@natron-dev-awesome/B1SW6Hbau)
- [X] I have followed the [contributing guidelines](https://github.com/NatronGitHub/Natron/blob/RB-2.4/CONTRIBUTING.md) to the best of my understanding
- [X] This feature is not on the [issue tracker](https://github.com/NatronGitHub/Natron/issues?q=is%3Abug+type%3Afeature+type%3Aidea) and is not a duplicate

### Provide a description of your feature request

In Preferences, the user will be able to set how curved node connectors will be. **0** will keep node connectors straight. The higher the number, the more the node connector will be curved. **20** will be the highest number that will make node connectors curved the most.

### Is this a realistically implementable feature?

- [X] My feature is realistically implementable
- [ ] My feature is NOT realistically implementable

### Can you contribute in creating this feature?

- [ ] I am unable, or not willing to contribute to creating this feature
- [X] I am able and willing to contribute to the creation of this feature

**Comment by devernay** [comment]
this is low priority, as it doesn't fix anything or add new functionality

**Comment by MKilo24** [comment]
> this is low priority, as it doesn't fix anything or add new functionality

Yes, sure. This is not a major feature anyways so this is not that much a priority.
